frontend/src/pages/admin/Template/utils.js

```diff
@@ -291,6 +291,36 @@ function prepareTemplateForEdit (template) {
     return result
 }
 
+/**
+ * Parse .env file data into an object
+ * NOTES:
+ * * Variable expansion is not supported
+ * * Comments are not supported
```

Comment:
```suggestion
 * * Comments are not supported
```
Is that strictly true? I think the regex on line 308 means it will ignore lines that start with a comment character.

Reply:
In fairness, it should say comments are not imported / maintained like a regular .env file would be (they are discarded)